internal/app/network/project.go

```diff
@@ -41,7 +41,16 @@ func NewProject(client *http.Client, payload ProjectPayload) ([]byte, error) {
 		return nil, &app.Error{Err: err, Kind: app.InternalError}
 	}
 
-	resp, err := client.Post(fmt.Sprintf("%s/project/generate", config.GenBaseUrl()), "application/json", &body)
+	req, err := http.NewRequest("POST", fmt.Sprintf("%s/project/generate", config.GenBaseUrl()), &body)
+
+	if err != nil {
+		return nil, &app.Error{Err: err, Kind: app.InternalError}
+	}
+
+	req.Header.Set("Content-Type", "application/json")
+	req.Header.Set("User-Agent", "Ktor CLI")
```

Comment:
You can add the version on here like "Ktor CLI/1.0" so we can keep track of what versions people are working with.

Reply:
Btw, `Ktor CLI` seems not a valid user agent as [spec doesn’t allow spaces](https://httpwg.org/specs/rfc9110.html#field.user-agent).
See https://github.com/ktorio/ktor/pull/4412